### PR TITLE
Adding the docs for CsTeamsEventsPolicy cmdlets

### PR DESCRIPTION
**Pull request by musingh1**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:77]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (get-itips) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:193]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+Required: True
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EventAccessType
+This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+Set this to EveryoneInCompanyExcludingGuests to disable a user from creating public webinars
```
Comment (get-itips) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:58]: What are all the possible values of this parameter? Everyone and EveryoneInCompanyExcludingGuests? please specify

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+Returns information about the Teams Events policy. TeamsEventsPolicy is used to configure options for customizing Teams Events experiences. 
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsEventsPolicy
+```
+
+Return information for all Teams Events policies available for use in the tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:35]:
```suggestion
Returns information for all Teams Events policies available for use in the tenant.

```

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+### Example 2
+```powershell
+PS C:\> Get-CsTeamsEventsPolicy -Identity Global
+```
+
+Return information for Teams Events policy with identity "Global".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:42]:
```suggestion
Returns information for Teams Events policy with identity "Global".

```

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+Unique identifier assigned to the Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:62]:
```suggestion
Unique identifier assigned to the Teams Events policy.

```

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md (hunk 1 above)
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:77]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/grant-csteamseventspolicy
+schema: 2.0.0
+---
+
+# Grant-CsTeamsEventsPolicy
+
+## SYNOPSIS
+Assigns Teams Events policy to a user, group of users or the entire tenant. Note that this policy is currently still in preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:11]:
```suggestion
Assigns Teams Events policy to a user, group of users, or the entire tenant. Note that this policy is currently still in preview.

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+```
+
+### GrantToGroup
+```
+Grant-CsTeamsEventsPolicy [-PassThru] [[-PolicyName] <String>] 
+ [-Group] <String> [-Rank <Int32>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Assigns Teams Events policy to a user, group of users or the entire tenant.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:33]:
```suggestion
Assigns Teams Events policy to a user, group of users, or the entire tenant.

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 9]
```diff
@@ -0,0 +1,204 @@
...
+TeamsEventsPolicy is used to configure options for customizing Teams Events experiences. 
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Grant-CsTeamsEventsPolicy -Identity "user1@contoso.com" -Policy DisablePublicWebinars
+```
+
+The command shown in Example 1 assigns the per-user Teams events policy, DisablePublicWebinars, to the user with the user principal name (UPN) "user1@contoso.com".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:43]:
```suggestion
The command shown in Example 1 assigns the per-user Teams Events policy, DisablePublicWebinars, to the user with the user principal name (UPN) "user1@contoso.com".

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 10]
```diff
@@ -0,0 +1,204 @@
...
+
+### Example 2
+```powershell
+PS C:\> Grant-CsTeamsEventsPolicy -Identity "user1@contoso.com" -Policy $null
+```
+
+The command shown in Example 2 revokes the per-user Teams events policy for the user with the user principal name (UPN) "user1@contoso.com". As a result, the user will be managed by the global Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:50]:
```suggestion
The command shown in Example 2 revokes the per-user Teams Events policy for the user with the user principal name (UPN) "user1@contoso.com". As a result, the user will be managed by the global Teams Events policy.

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 11]
```diff
@@ -0,0 +1,204 @@
...
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsEventsPolicy -Group "sales@contoso.com" -Rank 10 -Policy DisablePublicWebinars
+```
+
+The command shown in Example 3 assigns the Teams events policy, DisablePublicWebinars, to the members of the group "sales@contoso.com".
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:57]:
```suggestion
The command shown in Example 3 assigns the Teams Events policy, DisablePublicWebinars, to the members of the group "sales@contoso.com".

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 12]
```diff
@@ -0,0 +1,204 @@
...
+### -Identity
+Specifies the identity of the target user. Acceptable values include:
+
+Example: jphillips@contoso.com
+
+Example: sip:jphillips@contoso.com
+
+Example: 98403f08-577c-46dd-851a-f0460a13b03d
+
+Use the "Global" Identity if you wish to set the policy for the entire tenant
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:115]:
```suggestion
Use the "Global" Identity if you wish to set the policy for the entire tenant.

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 13]
```diff
@@ -0,0 +1,145 @@
...
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/new-csteamseventspolicy
+schema: 2.0.0
+---
+
+# New-CsTeamsEventsPolicy
+
+## SYNOPSIS
+This cmdlet allows you to create a new TeamsEventsPolicy instance and set it's properties. Note that this policy is currently still in preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:11]:
```suggestion
This cmdlet allows you to create a new TeamsEventsPolicy instance and set its properties. Note that this policy is currently still in preview.

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 14]
```diff
@@ -0,0 +1,145 @@
...
+```powershell
+PS C:\> New-CsTeamsEventsPolicy -Identity DisableWebinars -AllowWebinars Disabled
+```
+
+The command shown in Example 2 creates a new per-user Teams Events policy with the Identity DisableWebinars. This policy disables a user from creating webinars.
+
+## PARAMETERS
+
+### -AllowWebinars
+This setting governs if a user can create Webinars using Teams Events. Set this to Disabled to disable a user from creating webinars
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:42]:
```suggestion
This setting governs if a user can create webinars using Teams Events. Set this to **Disabled** to disable a user from creating webinars.

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 15]
```diff
@@ -0,0 +1,145 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
+Enables administrators to provide explanatory text to accompany a Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:88]:
```suggestion
Enables administrators to provide explanatory text to accompany a Teams Events policy.

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 16]
```diff
@@ -0,0 +1,145 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+Unique identifier assigned to the Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:103]:
```suggestion
Unique identifier assigned to the Teams Events policy.

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 17]
```diff
@@ -0,0 +1,145 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:134]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/Remove-CsTeamsEventsPolicy.md [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+Deletes a previously created TeamsEventsPolicy. Any users with no explicitly assigned policies will then fall back to the default policy in the organization. You cannot delete the global policy from the organization.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-CsTeamsEventsPolicy -Identity DisablePublicWebinars
+```
+
+In the example shown above, the command will delete the DisablePublicWebinars policy from the organization's list of policies.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsEventsPolicy.md:30]:
```suggestion
In this example, the command will delete the DisablePublicWebinars policy from the organization's list of policies.

```

File: teams/teams-ps/teams/Remove-CsTeamsEventsPolicy.md [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsEventsPolicy.md:81]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 20]
```diff
@@ -0,0 +1,138 @@
...
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/set-csteamseventspolicy
+schema: 2.0.0
+---
+
+# Set-CsTeamsEventsPolicy
+
+## SYNOPSIS
+This cmdlet allows you to configure options for customizing Teams events experiences. Note that this policy is currently still in preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:11]:
```suggestion
This cmdlet allows you to configure options for customizing Teams Events experiences. Note that this policy is currently still in preview.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 21]
```diff
@@ -0,0 +1,138 @@
...
+
+## SYNTAX
+
+```
+Set-CsTeamsEventsPolicy [-AllowWebinars <String>] [-Description <String>] [-EventAccessType <String>]
+ [[-Identity] <String>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+User-level policy for tenant admin to configure options for customizing Teams events experiences. Use this cmdlet to update an existing policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:21]:
```suggestion
User-level policy for tenant admin to configure options for customizing Teams Events experiences. Use this cmdlet to update an existing policy.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 22]
```diff
@@ -0,0 +1,138 @@
...
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Set-CsTeamsEventsPolicy -Identity Global -AllowWebinars Disabled
+```
+
+The command shown in Example 1 sets the value of the Default (Global) Events Policy in the organization to disable webinars, and leaves all other parameters the same.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:30]:
```suggestion
The command shown in Example 1 sets the value of the default (Global) Events Policy in the organization to disable webinars and leaves all other parameters the same.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 23]
```diff
@@ -0,0 +1,138 @@
...
+
+## PARAMETERS
+
+### -AllowWebinars
+This setting governs if a user can create Webinars using Teams Events.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:35]:
```suggestion
This setting governs if a user can create webinars using Teams Events.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 24]
```diff
@@ -0,0 +1,138 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
+Enables administrators to provide explanatory text to accompany a Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:65]:
```suggestion
Enables administrators to provide explanatory text to accompany a Teams Events policy.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 25]
```diff
@@ -0,0 +1,138 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Identity
+Unique identifier assigned to the Teams events policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:96]:
```suggestion
Unique identifier assigned to the Teams Events policy.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 26]
```diff
@@ -0,0 +1,138 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:127]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/teams.md [hunk 27]
```diff
@@ -65,6 +68,9 @@ Returns information about the Teams Media Logging policy.
 ### [Get-TeamTargetingHierarchyStatus](Get-TeamTargetingHierarchyStatus.md)
 {{Manually Enter Get-TeamTargetingHierarchyStatus Description Here}}
 
+### [Grant-CsTeamsEventsPolicy](Grant-CsTeamsEventsPolicy.md)
+Assigns Teams Events policy to a user, group of users or the entire tenant. Note that this policy is currently still in preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/teams.md:72]:
```suggestion
Assigns Teams Events policy to a user, group of users, or the entire tenant. Note that this policy is currently still in preview.
```

File: teams/teams-ps/teams/teams.md [hunk 28]
```diff
@@ -65,6 +68,9 @@ Returns information about the Teams Media Logging policy.
...
+
 ### [Grant-TeamsEnhancedEncryptionPolicy](Grant-TeamsEnhancedEncryptionPolicy.md)
 Cmdlet to assign a specific Teams enhanced encryption Policy to a user.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/teams.md:75]:
```suggestion
Cmdlet to assign a specific Teams enhanced encryption policy to a user.
```

File: teams/teams-ps/teams/teams.md [hunk 29]
```diff
@@ -113,6 +125,9 @@ Use this cmdlet to remove an existing Teams enhanced encryption policy.
 ### [Remove-TeamTargetingHierarchy](Remove-TeamTargetingHierarchy.md)
 {{Manually Enter Remove-TeamTargetingHierarchy Description Here}}
 
+### [Set-CsTeamsEventsPolicy](Set-CsTeamsEventsPolicy.md)
+Allows you to configure options for customizing Teams events experiences. Note that this policy is currently still in preview.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/teams.md:129]:
```suggestion
Allows you to configure options for customizing Teams Events experiences. Note that this policy is currently still in preview.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @musingh1, kindly commit the suggested changes. Many thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after the suggested changes.

**Comment by yogkumgit** [comment]
@musingh1 Please implement the suggested changes. Thanks

File: teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md (hunk 1 above)
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsEventsPolicy.md:77]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md [hunk 30]
```diff
@@ -0,0 +1,210 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Grant-CsTeamsEventsPolicy.md:199]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 31]
```diff
@@ -0,0 +1,155 @@
...
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EventAccessType
+This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+Possible values are:
+ - **Everyone**, Enables creating events to allow in tenant + guests + federated + anonymous (external to tenant) users to register and join the event
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:64]:
```suggestion
 - **Everyone**, Enables creating events to allow in-tenant, guests, federated, and anonymous (external to the tenant) users to register and join the event

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 32]
```diff
@@ -0,0 +1,155 @@
...
+ - **EveryoneInCompanyExcludingGuests**, Enables creating events to allow only in tenant users to register and join the event
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:65]:
```suggestion
 - **EveryoneInCompanyExcludingGuests**, Enables creating events to allow only in-tenant users to register and join the event

```

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 33]
```diff
@@ -0,0 +1,155 @@
...
+
+The command shown in Example 2 creates a new per-user Teams Events policy with the Identity DisableWebinars. This policy disables a user from creating webinars.
+
+## PARAMETERS
+
+### -AllowWebinars
+This setting governs if a user can create webinars using Teams Events. 
+Possible values are:
+ - **Enabled**, Enables creating webinars
+ - **Disabled**, Disables creating webinars
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:46]:
```suggestion
 - **Enabled**: Enables creating webinars.
 - **Disabled**: Disables creating webinars.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 34]
```diff
@@ -0,0 +1,145 @@
...
+
+The command shown in Example 1 sets the value of the Default (Global) Events Policy in the organization to disable webinars, and leaves all other parameters the same.
+
+## PARAMETERS
+
+### -AllowWebinars
+This setting governs if a user can create webinars using Teams Events.
+Possible values are:
+ - **Enabled**, Enables creating webinars
+ - **Disabled**, Disables creating webinars
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:38]:
```suggestion
 - **Enabled**: Enables creating webinars.
 - **Disabled**: Disables creating webinars.

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 35]
```diff
@@ -0,0 +1,145 @@
...
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EventAccessType
+This setting governs which users can access the event registration page or the event site to register. It also governs which user type is allowed to join the session/s in the event. 
+Possible values are:
+ - **Everyone**, Enables creating events to allow in tenant + guests + federated + anonymous (external to tenant) users to register and join the event
+ - **EveryoneInCompanyExcludingGuests**, Enables creating events to allow only in tenant users to register and join the event
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:87]:
```suggestion
 - **Everyone**: Enables creating events to allow in-tenant, guests, federated, and anonymous (external to the tenant) users to register and join the event.
 - **EveryoneInCompanyExcludingGuests**: Enables creating events to allow only in-tenant users to register and join the event.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@musingh1, a few more suggested changes, please.

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@musingh1 Gentle reminder. Thanks

**Comment by yogkumgit** [comment]
@musingh1 Gentle reminder.

**Comment by yogkumgit** [comment]
@musingh1 Please implement the suggested changes so that merge can be initiated. In the event of no response we may proceed to close this PR. Thanks

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@musingh1, thank you for committing the suggested changes, but you seem to have missed one. Could you also commit that one, please? Many thanks.

**Comment by musingh1** [comment]
> @musingh1, thank you for committing the suggested changes, but you seem to have missed one. Could you also commit that one, please? Many thanks.

Done!

**Comment by yogkumgit** [comment]
@JohanFreelancer9 @get-itips

**Comment by yogkumgit** [comment]
@JohanFreelancer9 @get-itips Gentle reminder. Please approve.

**Review by get-itips — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@pbafna03 
Hi!
I sent you email with a few questions to answer before I can merge this PR.
Thanks!

**Pull request merged**